pkg/trafficrouting/network/ingress/ingress.go

```diff
@@ -76,47 +78,48 @@ func NewIngressTrafficRouting(client client.Client, conf Config) (network.Networ
 // Initialize verify the existence of the ingress resource and generate the canary ingress
 func (r *ingressController) Initialize(ctx context.Context) error {
 	ingress := &netv1.Ingress{}
-	err := r.Get(ctx, types.NamespacedName{Namespace: r.conf.RolloutNs, Name: r.conf.TrafficConf.Name}, ingress)
```

Comment:
1. plz update Initialize func to reflect that Initialize only verify the existence of the ingress resource
2. if the ingress resource is generated in EnsureRoutes, will the rollout controller stuck in the Initializing condition?